File: src/main/java/com/github/penguin418/todolist/model/dto/TodoDto.java
package com.github.penguin418.todolist.model.dto;

import com.github.penguin418.todolist.model.constant.TodoState;
import lombok.Data;

import java.time.LocalDateTime;

@Data
public class TodoDto {
    private Long todoId;
    private Long userId;
    private String userNickname;
    private String todoContent;
    private TodoState todoState;
    private Long createdBy;
    private LocalDateTime createdDatetime;
    private Long modifiedBy;
    private LocalDateTime modifiedDatetime;
}

File: src/main/java/com/github/penguin418/todolist/model/dto/TodoDto.kt
package com.github.penguin418.todolist.model.dto

import com.github.penguin418.todolist.model.constant.TodoState
import java.time.LocalDateTime

data class TodoDto (
    @JvmField val todoId: Long?,
    @JvmField val userId: Long?,
    @JvmField val userNickname: String?,
    @JvmField val todoContent: String?,
    @JvmField val todoState: TodoState?,
    @JvmField val createdBy: Long?,
    @JvmField val createdDatetime: LocalDateTime?,
    @JvmField val modifiedBy: Long?,
    @JvmField val modifiedDatetime: LocalDateTime?
)
